cards/faq-accordion/component.js

```diff
@@ -13,11 +13,18 @@ class faq_accordionCardComponent extends BaseCard['faq-accordion'] {
    */
   dataForRender(profile) {
     const linkTarget = AnswersExperience.runtimeConfig.get('linkTarget') || '_top';
+    let profileAnswer = '';
+    if (profile.answerV2 && profile.answerV2.html) {
+      profileAnswer = profile.answerV2.html;
+    } else if (profile.answerV2 && profile.answerV2.json) {
+      console.warn('JSON is not supported, please switch to HTML.')
```

Comment:
nit: I think we should say use the word 'convert' rather than 'switch' because the feature on the backend uses the word 'convert' as well

Reply:
updated in most recent commit